transformers/storage/repository.go

```diff
@@ -257,8 +258,37 @@ func (repository *MakerStorageRepository) GetFlopBidIds(contractAddress string)
 	return bidIds, err
 }
 
+func (repository *MakerStorageRepository) GetAuthUsers(contractAddress string) ([]string, error) {
+	contractAddressID, addressErr := repository.GetOrCreateAddress(contractAddress)
+	if addressErr != nil {
+		return nil, addressErr
+	}
+	var userAddresses []string
+	selectErr := repository.db.Select(&userAddresses, `
+		SELECT addresses.address
+		FROM maker.rely
+		    LEFT JOIN public.addresses ON rely.usr = addresses.id
+		WHERE rely.address_id = $1
+		UNION
+		SELECT addresses.address
+		FROM maker.rely
+		    LEFT JOIN public.addresses ON rely.msg_sender = addresses.id
+		WHERE rely.address_id = $1
+		UNION
+		SELECT addresses.address
+		FROM maker.deny
+		    LEFT JOIN public.addresses ON deny.usr = addresses.id
+		WHERE deny.address_id = $1
+		UNION
+		SELECT addresses.address
+		FROM maker.deny
+		LEFT JOIN public.addresses ON deny.msg_sender = addresses.id
+		WHERE deny.address_id = $1`, contractAddressID)
```

Comment:
The only reason this query needs to combine more than just `rely.usr` and `deny.usr` is because we also want to find the `msg_sender` of the transaction that deployed the contract. Since we don't have a direct way of doing that (constructor doesn't emit an event), we have to infer it.

The `msg_sender` of the first `rely` event _must_ be the `msg_sender` who deployed the contract, because at that point no one else is authorized to call `rely` (or any other method with the `auth` modifier). Technically I didn't need to `UNION` with _all_ `rely`/`deny` `msg_sender`s (just the 1st `rely` event's), but I figured we might as well for redundancy's sake.

Also worth noting that these tables will likely be very small, so hopefully there's no reason to worry about performance.